addons/filebeat/locals.tf

```diff
@@ -0,0 +1,44 @@
+locals {
+  name = "filebeat"
+
+  default_helm_config = {
+    name                       = try(var.filebeat_extra_configs.name, local.name)
+    chart                      = try(var.filebeat_extra_configs.chart, local.name)
+    repository                 = try(var.filebeat_extra_configs.repository, "https://helm.elastic.co")
+    version                    = try(var.filebeat_extra_configs.version, "8.5.1")
+    namespace                  = try(var.filebeat_extra_configs.namespace, "logging")
+    create_namespace           = try(var.filebeat_extra_configs.create_namespace, true)
+    description                = "Filebeat helm Chart deployment configuration"
+    timeout                    = try(var.filebeat_extra_configs.timeout, "600")
+    lint                       = try(var.filebeat_extra_configs.lint, "false")
+    repository_key_file        = try(var.filebeat_extra_configs.repository_key_file, "")
+    repository_cert_file       = try(var.filebeat_extra_configs.repository_cert_file, "")
+    repository_username        = try(var.filebeat_extra_configs.repository_password, "")
```

Comment:
Take pull from `master` and update this `repository_username` variable's value here only.

Reply:
Done